charts/woodpecker/charts/agent/templates/role.yaml

```diff
@@ -1,4 +1,32 @@
 {{- if and (.Values.serviceAccount.create) (.Values.serviceAccount.rbac.create) -}}
+{{- if eq (.Values.env.WOODPECKER_BACKEND_K8S_NAMESPACE_PER_ORGANIZATION | toString) "true" -}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "woodpecker-agent.serviceAccountName" . }}
```

Comment:
This will create two cluster roles with the same name and might lead to overrides/confusions?
Why do we even need to duplicate the entire cluster role? Can we not conditionally include the additional permissions in case of `WOODPECKER_BACKEND_K8S_NAMESPACE_PER_ORGANIZATION` is enabled? The same applies to the role binding.